.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: go
 
+go_import_path: github.com/census-instrumentation/opencensus-service
+
 go:
   - 1.10.x
-
-before_script:
```

Comment:
I believe we want the Travis configuration to always run these scripts. Currently I don't see how they are run by Travis in combination with the Makefile